web/src/cypress/integration/rotations.ts

```diff
@@ -252,7 +252,9 @@ function testRotations(screen: ScreenFormat) {
         .clear()
         .type('5')
 
-      cy.get('button[type=submit]').click()
+      cy.get('button[type=submit]')
```

Comment:
Can we add something like `[data-cy=edit]` or maybe reference an aria label on the edit button? Selecting all submit buttons, then excluding `set-fav` seems fragile (e.g. if we add an another one)